### ISSUE TITLE
Option to play audio-only like in NewPipe.

### ISSUE THREAD

**Issue by Iaotle** [ReVancedTeam/revanced-suggestions#26]
Would save some bandwidth and battery power.

**Comment by oSumAtrIX** [comment]
Isn't that done automatically on background playback?

**Comment by GreenMan36** [comment]
YouTube uses surprisingly little data anyways. 144p should be quite good.
But Audio-Only would still be nice to have.

**Comment by oSumAtrIX** [comment]
> This entirely depends if YouTube provides separate streams for video and audio.

IIRC they do. This is why yt-dl can download off thr audio stream. Hence I assumed that background playback already only downloads audio.

**Comment by ZeloIsGay** [comment]
> Isn't that done automatically on background playback?

Yes. I don't have any hard evidence but I'd tested vanced's background playback bandwidth before and it was evident that background playback did not download the video content.

**Comment by MSOB7YY** [comment]
background playback in vanced didnt download any video streams, it just played the audio.
apps like YMusic are also capable of playing audio only so i think it's possible.

**Comment by oSumAtrIX** [comment]
So it is possible, already why suggest this as a seperate feature then?

**Comment by Iaotle** [comment]
I didn't know it was possible, would also be nice to have foreground play with audio only tho.

**Comment by oSumAtrIX** [comment]
> foreground play with audio only tho.

For what reason, are you going to stare at a black screen and listen to the audio? Simply go to the home screen or turn off your screen

**Comment by Lucaskyy** [comment]
It seems like you want a "stripped YTM" in YT. Even though this seems understandable, is there any value towards it why you cannot use YTM and do it that way?

**Comment by MSOB7YY** [comment]
> 
> For what reason, are you going to stare at a black screen and listen to the audio? Simply go to your phones home screen or turn off your screen

hmmm maybe saving data while checking recommendations or playlists?
i wouldn't do that tho, as 144p/240p already uses almost the same data.

**Comment by MulverineX** [comment]
Perhaps instead of having a no-music UI built-in, there could be a long-press action on videos in the app that lets you open links externally, and you can configure that to YouTube Music? Should be able to quickly back-arrow out of the app too and get back to YouTube.

**Comment by oSumAtrIX** [comment]
To be honest this is a feature I will not implement personally due to low interest/useless, but since ReVanced is open source I welcome anyone to do it if they like to )
